manifests/v/VentisMedia/MediaMonkey/5/VentisMedia.MediaMonkey.installer.yaml

```
@@ -1,20 +1,77 @@
+# Created using YamlCreate.ps1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
 PackageIdentifier: VentisMedia.MediaMonkey
 PackageVersion: 5
 MinimumOSVersion: 10.0.0.0
 FileExtensions:
+- 3gp
+- 3gpp
+- aac
+- adts
+- aif
+- aifc
+- aiff
+- alac
+- ape
+- apl
+- asf
+- asx
+- avi
+- cda
+- cue
+- divx
+- f4v
+- fla
+- flac
+- flv
+- ifo
+- isma
+- m1v
+- m2v
+- m3u
+- m3u8
+- m4a
+- m4b
+- m4p
+- m4v
+- mac
+- mkv
+- mmip
+- mov
 - mp3
+- mp4
+- mpc
+- mpe
+- mpeg
+- mpg
+- mpp
+- mpv
+- ogg
+- ogm
+- ogv
+- pla
+- pls
+- wav
+- webm
+- wm
+- wma
+- wmv
+- xvid
 InstallModes:
-  - interactive
-  - silent
-  - silentWithProgress
+- interactive
+- silent
+- silentWithProgress
 Installers:
-  - Architecture: x86
-    InstallerType: inno
-    InstallerUrl: https://www.mediamonkey.com/sw/MediaMonkey_5.0.0.2338.exe
-    InstallerSha256: 30545D90F20E5F6F402025798171D3F1D8025910F81F9B456E8EA94E9D4264EA
-    Scope: machine
-    InstallerLocale: en-US
-    UpgradeBehavior: install
+- InstallerLocale: en-us
+  Architecture: x86
+  InstallerType: inno
+  Scope: user
```

```suggestion
  Scope: machine
```
Application installs in `C:\Program Files (x86)\MediaMonkey 5`.